_layouts/integration.html

```diff
@@ -73,6 +86,36 @@ <h2>API support</h2>
   </details>
 </section>
 
+{% if page.quality_estimation_integrations.size > 0 %}
+<section id="qe-integrations">
+  <h2>Quality estimation</h2>
+  <details>
+    <summary>
+      <strong>{{ page.quality_estimation_integrations| size }}</strong> Quality estimation API integrations are available for {{ page.title }}.
+      <p class="preview hint">
+        {{page.quality_estimation_integrations | slice: 0, 5 | map: 'name' | join: ', ' }}
+        {% if page.quality_estimation_integrations > 5 %}, …{% endif %}
+      </p>
+    </summary>
+    <ul>
+    {% for qe in page.quality_estimation_integrations %}
+      <li>
+        {% if qe.plugin %}
```

Comment:
```
<a href="/{{ qe.slug }}">{{ qe.name }}</a> {% if qe.plugin %}(plugin){% endif %}
```

Should probably change it for MT APIs too.

For QE, we'll need a "connector" option too.  Many of the de facto integrations are connectors, not integrations or plugins.